src/error.js

```diff
@@ -0,0 +1,15 @@
+import { errorTypes } from "./constants.js";
+
+export const createError = (type, key, value) => {
+  if (typeof type !== "String" || !errorTypes[type]) {
```

Comment:
Is it necessary to check if the type is String? I think it could work without it.

Reply:
I checked it and it is not necessary. Thanks!